unity-environment/ProjectSettings/TagManager.asset

```diff
@@ -25,6 +25,7 @@ TagManager:
   - switchOff
   - pyramid
   - switchOn
+  - stone
```

Comment:
So many tags... Can we have themn more generic like `object0`, `object1`, `enemy0`...